+++
title = "Mapping the regulatory landscape of auditory hair cells from single-cell multi-omics data"
date = "2021-04-09"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["S Wang", "M P. Lee", "S Jones", "__J Liu__", "J Waldhaus"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "In *Genome Research*"
publication_short = "In *Genome Res*"

# Abstract and optional shortened version.
abstract = "Auditory hair cells transduce sound to the brain and in mammals these cells reside together with supporting cells in the sensory epithelium of the cochlea, called the organ of Corti. To establish the organ's delicate function during development and differentiation, spatiotemporal gene expression is strictly controlled by chromatin accessibility and cell type-specific transcription factors, jointly representing the regulatory landscape. Bulk-sequencing technology and cellular heterogeneity obscured investigations on the interplay between transcription factors and chromatin accessibility in inner ear development. To study the formation of the regulatory landscape in hair cells, we collected single-cell chromatin accessibility profiles accompanied by single-cell RNA data from genetically labeled murine hair cells and supporting cells after birth. Using an integrative approach, we predicted cell type-specific activating and repressing functions of developmental transcription factors. Furthermore, by integrating gene expression and chromatin accessibility datasets, we reconstructed gene regulatory networks. Then, using a comparative approach, 20 hair cell-specific activators and repressors, including putative downstream target genes, were identified. Clustering of target genes resolved groups of related transcription factors and was utilized to infer their developmental functions. Finally, the heterogeneity in the single-cell data allowed us to spatially reconstruct transcriptional as well as chromatin accessibility trajectories, indicating that gradual changes in the chromatin accessibility landscape were lagging behind the transcriptional identity of hair cells along the organ's longitudinal axis. Overall, this study provides a strategy to spatially reconstruct the formation of a lineage specific regulatory landscape using a single-cell multi-omics approach."
abstract_short = ""

# Summary. An optional shortened abstract.
summary = ""

# Digital Object Identifier (DOI)
doi = "10.1101/gr.271080.120"

# Is this a featured publication? (true/false)
feature = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
projects = []

# Links (optional).
url_pdf = ""
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
url_custom = []

# Tags (optional).
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = []

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = false

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = ""

+++


